.bin/phraseapp/wd-project.rb

```diff
@@ -89,7 +90,13 @@ def json_generate_from_worktree
     @log.info('Gathering keys from local worktree into a temporary JSON file...')
 
     worktree_keys = @translation_builder.get_all_keys
-    h = Hash[worktree_keys.map { |x| [x, ''] }]
+
+    keys = []
+    worktree_keys.each do |key|
+      keys.push(key.sub(@locale_prefix, ''))
+    end
+
+    h = Hash[keys.map { |x| [x, ''] }]
```

Comment:
You can avoid the additional `.each` loop by modifying this line only, like this:
```ruby
h = Hash[keys.map { |x| [x.sub(@locale_prefix, ''), ''] }]
```